internal/docs/release.md

```diff
@@ -1,11 +1,10 @@
 # Releases
 
-We aim to do a release after each Sprint (roughly, every 3 weeks).
-
 To do a release:
 
--   Do the prerelease checks until everything is passing.
-    -   Create a new branch (`git branch -B prerelease`). Running the script
+-   Make sure all tests pass localy with `runchecks.sh` and Travis is green.
```

Comment:
Insert something here about announcing to the team that you're doing a release -- checkins should be frozens during this process.

Reply:
Done